lib/active_shipping/carriers/usps.rb

```diff
@@ -154,6 +154,11 @@ class USPS < Carrier
       "WS" => "Western Samoa"
     }
 
+    TRACKING_ODD_COUNTRY_NAMES = {
```

Comment:
There's lots of other hashes like this in the same place so I thought I would do it a consistent way, and there should only be a few of them.